resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -126,7 +126,7 @@ If you prefer to run just the integration tests directly, then you can customise
 properties to Maven.  For example:
 
     mvn integration-test -am -pl :spark-kubernetes-integration-tests_2.12 \
-                            -Pkubernetes -Pkubernetes-integration-tests \ 
+                            -Pkubernetes -Pkubernetes-integration-tests \
```

Comment:
Removed this tailing white space in this PR so that we can copy, paste, edit and run.